mlir/include/mlir/Dialect/Rock/Tuning/GridwiseGemmParams.h

```diff
@@ -203,6 +207,8 @@ class PopulateParams : public BasePopulateParams<InitParamsNonXDL> {
 
   const InitParams &getUniversalParameters() const;
```

Comment:
Do we know why this is returning an `InitParams` and not one of its subtypes? That would get rid of the need for `getUniversalKPack()`